File: Readio/src/main/java/ninegle/Readio/book/dto/booksearch/BookSearchResponseDto.java
package ninegle.Readio.book.dto.booksearch;

import java.math.BigDecimal;

import lombok.Builder;


@Builder
public record BookSearchResponseDto(long id, String name, String image, String categoryMajor, String categorySub,
									String authorName, BigDecimal rating) {

}

File: Readio/src/main/java/ninegle/Readio/book/dto/booksearch/BookSearchResponseDto.kt
package ninegle.Readio.book.dto.booksearch

import java.math.BigDecimal

data class BookSearchResponseDto(
    val id: Long?,
    val name: String,
    val image: String,
    val categoryMajor: String,
    val categorySub: String,
    val authorName: String,
    val rating: BigDecimal
)
